<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Config</class>
 <widget class="QDialog" name="Config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>530</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/main icon</normaloff>:/icons/main icon</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="tabPosition">
        <enum>QTabWidget::North</enum>
       </property>
       <property name="tabShape">
        <enum>QTabWidget::Rounded</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="documentMode">
        <bool>false</bool>
       </property>
       <property name="tabsClosable">
        <bool>false</bool>
       </property>
       <property name="movable">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="tabSerial">
        <attribute name="title">
         <string>Settings</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>List of serial ports:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTreeWidget" name="treeWidget">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <column>
              <property name="text">
               <string>Port</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="buttonRefresh">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/icons/resources/icons/arrow_refresh.png</normaloff>:/icons/resources/icons/arrow_refresh.png</iconset>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>Serial</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLabel" name="label_9">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>Select a port from the left</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QFormLayout" name="formLayout">
                <property name="fieldGrowthPolicy">
                 <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
                </property>
                <item row="0" column="0">
                 <widget class="QComboBox" name="boxTTY">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>100</width>
                    <height>0</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>COM/tty</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QComboBox" name="boxBaudrate">
                  <property name="minimumSize">
                   <size>
                    <width>100</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="currentText">
                   <string/>
                  </property>
                  <property name="maxVisibleItems">
                   <number>20</number>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>Baudrate</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLineEdit" name="editBaudrate">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>100</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="toolTip">
                   <string>Future version</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>Custom baudrate</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QComboBox" name="boxDatabits">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>100</width>
                    <height>0</height>
                   </size>
                  </property>
                  <item>
                   <property name="text">
                    <string>8</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>7</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>6</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>5</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>Databits</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0">
                 <widget class="QComboBox" name="boxStopbits">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>100</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="toolTip">
                   <string>Future version</string>
                  </property>
                  <item>
                   <property name="text">
                    <string>1</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>1.5</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>2</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="4" column="1">
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>Stopbits</string>
                  </property>
                 </widget>
                </item>
                <item row="5" column="0">
                 <widget class="QComboBox" name="boxParity">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>100</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="toolTip">
                   <string>Future version</string>
                  </property>
                  <item>
                   <property name="text">
                    <string>None</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Even</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Odd</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Mark</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Space</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="5" column="1">
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>Parity</string>
                  </property>
                 </widget>
                </item>
                <item row="6" column="0">
                 <widget class="QComboBox" name="boxFlowctrl">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>100</width>
                    <height>0</height>
                   </size>
                  </property>
                  <item>
                   <property name="text">
                    <string>None</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>CTS/RTS</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Xon/Xoff</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="6" column="1">
                 <widget class="QLabel" name="label_11">
                  <property name="text">
                   <string>Flow control</string>
                  </property>
                 </widget>
                </item>
                <item row="7" column="0">
                 <widget class="QComboBox" name="boxReconnect">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>100</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="toolTip">
                   <string>Future version</string>
                  </property>
                  <item>
                   <property name="text">
                    <string>No</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Immediately</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>After 10 seconds</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="7" column="1">
                 <widget class="QLabel" name="label_10">
                  <property name="text">
                   <string>Auto reconnect</string>
                  </property>
                 </widget>
                </item>
                <item row="8" column="0">
                 <widget class="QPushButton" name="buttonDefaults">
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="resources.qrc">
                    <normaloff>:/icons/resources/icons/arrow_undo.png</normaloff>:/icons/resources/icons/arrow_undo.png</iconset>
                  </property>
                  <property name="autoDefault">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>Server</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <layout class="QFormLayout" name="formLayout_2">
                <property name="fieldGrowthPolicy">
                 <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
                </property>
                <item row="0" column="0">
                 <widget class="QSpinBox" name="boxTCPPort">
                  <property name="minimum">
                   <number>1025</number>
                  </property>
                  <property name="maximum">
                   <number>65535</number>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>TCP port number</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0" colspan="2">
                 <widget class="QCheckBox" name="checkOnlyLocal">
                  <property name="text">
                   <string>Accept only localhost connections</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOk">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonApply">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
